.ci/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -88,7 +93,7 @@ set PATH=%CUDA_PATH%\bin;%CUDA_PATH%\libnvvp;%PATH%
 :cuda_build_end
 
 set DISTUTILS_USE_SDK=1
-set PATH=%TMP_DIR_WIN%\bin;%PATH%
+set PATH=%TMP_DIR_WIN%\bin;C:\Program Files\CMake\bin;%PATH%
```

Comment:
Wasnt' cmake already added to path by CHOCO?

Reply:
Because conda has been activated above, and its PATH override system path..